src/tlo/methods/healthsystem.py

```diff
@@ -1313,8 +1310,8 @@ def schedule_hsi_event(
         # Check that priority is in valid range
         assert priority >= 0
 
-        # If priority of HSI_Event lower than the lowest one considered, ignore event in scheduling
-        if priority > self.lowest_priority_considered:
+        # If priority of HSI_Event lower than the lowest one considered, ignore event in scheduling under mode 2
+        if (self.mode_appt_constraints == 2) and (priority > self.lowest_priority_considered):
```

Comment:
why link this to mode?

Reply:
To ensure that even if adopting a policy which assigns certain HSIs a priority > lowest_priority_considered, in the mode 1 phase these are still queued.